WordPress/Classes/ViewRelated/Views/WPRichText/WPRichTextImage.swift

```diff
@@ -88,13 +95,29 @@ open class WPRichTextImage: UIControl, WPRichTextMediaAttachment {
             onSuccess?(indexPath)
         }
 
-        let errorHandler: ((Error?) -> Void)? = { error in
+        let errorHandler: ((Error?) -> Void)? = { [weak self] error in
+            guard let self = self else { return }
+
+            self.retryImageView.delegate = self
+            self.addSubview(self.retryImageView)
+            self.retryImageView.leftAnchor.constraint(equalTo: self.layoutMarginsGuide.leftAnchor).isActive = true
+            self.retryImageView.topAnchor.constraint(equalTo: self.layoutMarginsGuide.topAnchor).isActive = true
+            self.retryImageView.rightAnchor.constraint(equalTo: self.layoutMarginsGuide.rightAnchor).isActive = true
+            self.retryImageView.bottomAnchor.constraint(equalTo: self.layoutMarginsGuide.bottomAnchor).isActive = true
```

Comment:
I'm not sure I follow, could you clarify please? 
`WPRichTextImage` doesn't use an xib but if you're referring to `ReaderRetryFailedImageView`'s xib, I don't see how it would add these constraints either.

Reply:
It's more of a question, really. I wasn't sure if it's possible. 🙂